spec/requests/cart_items/update_cart_item_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe '/cart_items', type: :request do
+  describe 'PATCH /cart_items/:id' do
+    subject(:request_call) do
+      patch "/cart_items/#{cart_item.id}", params: { cart_item: { quantity: quantity } }
+    end
+
+    let!(:user1) { create(:user) }
+    let!(:user2) { create(:user) }
```

Comment:
user2 should be created in the context for him

Reply:
fixed